packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,13 @@
 
     &__button {
         align-self: flex-start;
+        display: flex;
+        flex-flow: row wrap;
+        white-space: inherit;
+        height: fit-content;
+        max-height: 7rem;
```

Comment:
Made height to be `fit-content` and added max height to fix maximum height:

### Without fit-content:
<img width="453" alt="Screenshot 2022-07-18 at 3 17 47 PM" src="https://user-images.githubusercontent.com/103016120/179462234-5993d302-9287-48a2-8978-7df8ef9ae775.png">

### With fit-content:
<img width="478" alt="Screenshot 2022-07-18 at 3 18 05 PM" src="https://user-images.githubusercontent.com/103016120/179462249-e3398280-85c9-4d8e-9c48-587d3fee22e6.png">